GOVERNANCE.md

```diff
@@ -0,0 +1,130 @@
+# Jujutsu Governance
+
+## Overview
+
+Jujutsu is an open source project, led, maintained and designed for a worldwide
+community. Anyone who is interested can join, contribute, and participate in the
+decision-making process. This document is intended to help you understand how
+you can do that.
+
+## Project roles
+
+There are two broadly defined roles in the Jujutsu project: **Maintainers** and
+**Contributors**.
+
+### Maintainers
+
+At the top of the hierarchy are the **Maintainers**. We consider maintainers to

```

Comment:
suggestion: I don't think we need to explicitly call out anything "hierarchical", as it seems like somewhat anti-collaborative wording (and, indeed, we added the Contributor role to recognize the valuable contributions of non-Maintainers).

suggestion: I think we should move the actual authority and responsibility up to the top here as the primary definition of a Maintainer, namely that they can conduct votes to resolve conflicts. @nasamuffin already noted below that it didn't belong in the list of Maintainer behaviors.

suggestion: Let's capitalize Maintainer everywhere (assuming that's appropriate?).